jsk_naoqi_robot/naoqieus/naoqi-interface.l

```diff
@@ -328,15 +344,18 @@
    (let ((ret (instance nao_interaction_msgs::SetAudioMasterVolumeRequest :init)))
      (ros::wait-for-service (format nil "~A/naoqi_driver/set_volume" group-namespace))
      (send ret :master_volume :data volume)
+     (setq master-volume (/ volume 100.0))
      (ros::service-call (format nil "~A/naoqi_driver/set_volume" group-namespace) ret)
      ))
   (:get-master-volume
    ()
    (let ((ret (instance nao_interaction_msgs::GetAudioMasterVolumeRequest :init))
-	 res)
+	 res volume)
      (ros::wait-for-service (format nil "~A/naoqi_driver/get_volume" group-namespace))
      (setq res (ros::service-call (format nil "~A/naoqi_driver/get_volume" group-namespace) ret))
-     (send res :master_volume :data)))
+     (setq volume (send res :master_volume :data))
+     (setq master-volume (/ volume 100.0))
+     vol))
```

Comment:
I think `vol` should be `volume`.

`send *ri* :get-master-volume` =>

```
/opt/ros/melodic/share/euslisp/jskeus/eus/Linux64/bin/irteusgl 0 error: unbound variable vol in (let ((ret (instance nao_interaction_msgs::getaudiomastervolumerequest :init)) res volume) (ros::wait-for-service (format nil "~A/naoqi_
driver/get_volume" group-namespace)) (setq res (ros::service-call (format nil "~A/naoqi_driver/get_volume" group-namespace) ret)) (setq volume (send res :master_volume :data)) (setq master-volume (/ volume 100.0)) vol)
```

Reply:
After modifying `vol` to `volume`, I confirmed this PR works well (Ubuntu 18.04, ROS melodic).

```
roslaunch jsk_pepper_startup jsk_pepper_startup.launch network_interface:=enp4s0 launch_joy:=false (this branch)
roslaunch naoqi_apps speech.launch (kochigami-develop)
```
```
roscd peppereus
roseus pepper-inteface.l
pepper-init t

2.irteusgl$ send *ri* :set-language "English"
t
3.irteusgl$ send *ri* :get-language
"English"
4.irteusgl$ send *ri* :set-master-volume 50
 #<nao_interaction_msgs::setaudiomastervolumeresponse #X55c4e7bd63f0>
5.irteusgl$ send *ri* :get-master-volume
50
6.irteusgl$ send *ri* :speak "Hello, Hello, Hello, Hello, Hello" :lang "Japanese" :wait t :volume 0.4
[ INFO] [1660552895.025133938]: subscribing speech_status
[ INFO] [1660552895.524033720]: subscribing speech_status
[ INFO] [1660552896.024239547]: subscribing speech_status
[ INFO] [1660552896.524247881]: subscribing speech_status
[ INFO] [1660552897.024213482]: subscribing speech_status
[ INFO] [1660552897.524022335]: subscribing speech_status
[ INFO] [1660552898.024220278]: subscribing speech_status
[ INFO] [1660552898.524135728]: subscribing speech_status
[ INFO] [1660552899.024391846]: subscribing speech_status
[ INFO] [1660552899.524035537]: subscribing speech_status
nil
7.irteusgl$ send *ri* :get-language
"Japanese"
8.irteusgl$ send *ri* :get-master-volume
40
9.irteusgl$ send *ri* :speak "Hello, Hello, Hello, Hello, Hello" :wait t :volume 0.4
[ INFO] [1660552929.352141912]: subscribing speech_status
[ INFO] [1660552929.851521853]: subscribing speech_status
[ INFO] [1660552930.351535601]: subscribing speech_status
[ INFO] [1660552930.851412365]: subscribing speech_status
[ INFO] [1660552931.351527093]: subscribing speech_status
[ INFO] [1660552931.851525856]: subscribing speech_status
[ INFO] [1660552932.351486681]: subscribing speech_status
[ INFO] [1660552932.851489413]: subscribing speech_status
[ INFO] [1660552933.351495158]: subscribing speech_status
nil
10.irteusgl$ send *ri* :speak "Hello, Hello, Hello, Hello, Hello" :wait t :volume 0.4 :lang "English"
[ INFO] [1660552947.863459217]: subscribing speech_status
[ INFO] [1660552948.362732784]: subscribing speech_status
```

Reply:
@kochigami thank you fixed